src/nrniv/nrnpy.cpp

```diff
@@ -372,21 +371,31 @@ static void* load_nrnpython_helper(const char* npylib) {
 	return handle;
 }
 
-#if !DARWIN
-static int digittoint(char c) {
-	return int(c - '0');
+int digit_to_int(char ch) {
```

Comment:
as `digittoint` doesnt exist on some platform, we though it would be simple to use definition from BSD implementation rather than doing ifdefs.